Java/other/BenjaminBulb.java

```diff
@@ -0,0 +1,25 @@
+//Link to problem: https://leetcode.com/problems/bulb-switcher/
```

Comment:
Add a problem description

Reply:
The last time I had created a PR (had to close it because of some glitch) I was asked to add a link to the problem instead of the description. If the description is what is needed I will add that.